scripts/benchmark_db/README.md

```diff
@@ -1,49 +1,114 @@
 # Forest benchmark db script
 
-This script is here to help with testing of Forest db backends and their
-configuration.
+This script was developed to help with testing of Forest db backends and their
+configuration; the script now also allows benchmarking ("daily" benchmarks) of
+Forest and Lotus snapshot import times (in min:sec) and validation times (in
+tipsets/min).
 
 ## Install dependencies
 
-You will need to install Ruby first. Then go into `benchmark_db` and type:
+[Install Ruby](https://www.ruby-lang.org/en/documentation/installation/) first.
+Then go into `/forest/scripts/benchmark_db` and execute:
 
 ```
 $ bundle install
 ```
 
+Note: depending upon your Ruby installation, it may be necessary to execute
+`gem install bundler` first. In case of any issues with "native extensions"
+during `bundle install` on a \*nix machine, it may also be necessary to execute
+`apt-get update && apt-get install -y build-essential`.
```

Comment:
I believe the most common missed package for building native extensions is `ruby-devel` (for RPM-based distros)

Reply:
Could you please clarify what changes you are suggesting with this comment?

Reply:
`apt-get update && apt-get install -y build-essential ruby-dev`